apps/client/lib/graphql-queries.ts

```diff
@@ -71,6 +71,7 @@ export const queries = {
       title
       fullName
       _id
+      isManagerOf
```

Comment:
This shouldn't be here if the feature isn't complete. Could you remove it and add it in a new branch if you're working on it?

Personally, I would also just call it 'managerOf' since 'is' means `boolean` to me

Reply:
done!

Reply:
you can check